tensorly/decomposition/candecomp_parafac.py

```diff
@@ -307,6 +328,9 @@ def parafac(tensor, rank, n_iter_max=100, init='svd', svd='numpy_svd',\
                     pseudo_inverse = pseudo_inverse*tl.dot(tl.conj(tl.transpose(factor)), factor)
             pseudo_inverse += Id
 
+            if mask is not None:
+                tensor = tensor*mask + tl.kruskal_to_tensor((weights, factors), mask=1-mask)
```

Comment:
Good catch -- we weren't using the mask before??

Reply:
It's part of the error calculation step now. I'll remove this.